src/main/java/uk/gov/companieshouse/search/api/logging/LoggingUtils.java

```diff
@@ -15,7 +15,8 @@
     public static final String ALPHABETICAL_SEARCH = "alphabetical";
     public static final String COMPANY_NAME = "company_name";
     public static final String COMPANY_NUMBER = "company_number";
-    public static final String DISSOLVED_SEARCH = "dissolved";
+    public static final String DISSOLVED_SEARCH_ALPHABETICAL = "dissolved - alphabetical";
+    public static final String DISSOLVED_SEARCH_BEST_MATCH = "dissolved - alphabetical";
```

Comment:
These are both the same?

Reply:
Good spot, they are meant to be different, will resolve now 👍